interpreter/jinko.rs

```diff
@@ -225,7 +229,16 @@ fn main() -> anyhow::Result<()> {
     }
     #[cfg(feature = "repl")]
     let result = args.input().map_or_else(
-        || Repl::new()?.launch(),
+        || {
+            if args.experimental {
+                match xrepl::repl() {
+                    Ok(_) => Ok((None, Context::new(Box::new(jinko::io_trait::JkStdReader)))),
```

Comment:
this needs a FIXME with a comment to explain what we're doing and that we should remove it